### PR TITLE
DOCSP-41975: Retrieve data

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41975
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41975-retrieve-data/read/retrieve/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ a couple comments

**Review by GromNaN — CHANGES_REQUESTED** [review]

File: source/includes/read/retrieve.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+// end-cursor
+
+// Finds and prints up to 5 documents with a "number_of_employees" value of 1000
+// start-modify
+$options = [
+    'limit' => 5,
+];
+
+$results = $collection->find(['number_of_employees' => 1000], $options);
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/includes/read/retrieve.php]: Just for emphasis on the options, but I added some code highlighting instead

File: source/includes/read/retrieve.php [hunk 2]
```diff
@@ -29,14 +29,12 @@
...
+    ['limit' => 5]
+);
 
 foreach ($results as $doc) {
     echo json_encode($doc) . "\n";
 }
-// end-modify
-?>
\ No newline at end of file
+// end-modify
```
Comment (GromNaN) [inline source/includes/read/retrieve.php:40]:
```suggestion
// end-modify

```

**Review by GromNaN — APPROVED** [review]

File: snooty.toml [hunk 3]
```diff
@@ -24,3 +24,4 @@ php-library = "MongoDB PHP Library"
 
 [constants]
 php-library = "MongoDB PHP Library"
+api = "https://www.mongodb.com/docs/php-library/current/reference"
```
Comment (GromNaN) [inline snooty.toml:27]:
```suggestion
api = "https://www.mongodb.com/docs/php-library/current/reference"

```

**Review by GromNaN — APPROVED** [review]

**Pull request merged**